scala/private/rule_impls.bzl

```diff
@@ -509,20 +576,34 @@ def _collect_jars_from_common_ctx(ctx,
       jars2labels = jars2labels,
       transitive_compile_jars = transitive_compile_jars)
 
-def _lib(ctx, base_classpath, non_macro_lib):
+def _lib(ctx,
+         base_classpath,
+         non_macro_lib,
+         unused_dependency_checker_mode = "off"):
```

Comment:
Can we remove defaults as much as possible on private methods? It makes it too easy to overlook them.